### ISSUE TITLE
Non-visual version of Soundcloud embed

### ISSUE THREAD

**Issue by OKNoah** [itteco/iframely#52]
There are two versions of the embed from sound cloud. One is `&visual=true`, which has the artwork cover the background.

The other is the normal, non-visual one, that has the artwork on the far left.

iFramely shows the visual one, but I can't see in the source code where it requests that or how to change it.

**Comment by iparamonau** [comment]
We take Soundcloud from their oEmbed endpoint
http://iframely.com/debug?uri=https%3A%2F%2Fsoundcloud.com%2Flittlesimz%2Flittle-simz-e-d-g-e-04-devour

It appears they prefer the visual player now. 

The quickest fix for you would be to add "?show_artwork=false" to their oEmbed endpoint address:
https://github.com/itteco/iframely/blob/master/lib/plugins/system/oembed/providers.json#L176

In next release, we'll try to add this option to your local the config file, in a similar way we do for Youtubes, Vimeos etc.

**Comment by OKNoah** [comment]
Thank you!

**Comment by iparamonau** [comment]
So we finally figured a way to handle this. Take a look at the [commit](https://github.com/itteco/iframely/commit/e9761117a4d5da9ba9c6409a9b0e45e6d804d683) to develop branch.

We added `ADD_OEMBED_PARAMS` array into config file to match regexps of oEmbed endpoint with extra get params. 

However, apparently SoundCloud seems to deprecate `&visual=true` params. At least it isn't in their docs any longer. Hope you will have better luck. From our side - mechanism is there to support adjustments.
